scripts/generate_readme.rb

```diff
@@ -0,0 +1,89 @@
+require 'optparse'
```

Comment:
I don't really have a strong opinion one way or the other, but noting that the `changelog_check.rb` is executable (`chmod +x`'d). Should we be consistent in how these scripts are run?

Reply:
I think that chmod'ing should be for things that don't have an extension. Based on the name it's a ruby file so it seems natural to me to call it via `ruby` or `irb -r`